src/canvas-extractor/tests/graphql/unit/assignments/test_extractor_assignments.py

```diff
@@ -0,0 +1,50 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
@sgonzalezgap it appears that there is an entire extra directory of tests, called `unit`. These conflict with the main test files. Is it safe to just delete this directory?